dbfit-java/core/src/main/java/dbfit/util/FitNesseTestHost.java

```diff
@@ -9,6 +10,7 @@
 
     static {
         TypeAdapter.registerParseDelegate(BigDecimal.class, BigDecimalParseDelegate.class);
+        TypeAdapter.registerParseDelegate(BigInteger.class, BigIntegerParseDelegate.class);
         TypeAdapter.registerParseDelegate(java.sql.Date.class, SqlDateParseDelegate.class);
         TypeAdapter.registerParseDelegate(java.sql.Timestamp.class, SqlTimestampParseDelegate.class);
         TypeAdapter.registerParseDelegate(java.sql.Time.class, SqlTimeParseDelegate.class);
```

Comment:
There are some predefined type adapters in [`fit.TypeAdapter`](https://github.com/unclebob/fitnesse/blob/master/src/fit/TypeAdapter.java); for anything else - a default one is used supporting String, Date and objects with `parse` method. And for anything else - usually a custom parse delegate is needed (otherwise an error is failed when trying to parse the values).